app/models/ems_event.rb

```diff
@@ -72,6 +72,10 @@ def self.add_openstack(ems_id, event)
     add(ems_id, ManageIQ::Providers::Openstack::CloudManager::EventParser.event_to_hash(event, ems_id))
   end
 
+  def self.add_openstack_network(ems_id, event)
+    add(ems_id, ManageIQ::Providers::Openstack::NetworkManager::EventParser.event_to_hash(event, ems_id))
+  end
```

Comment:
Note to self:  these methods need to be in the providers' code bases (future refactor)